### PR TITLE
Logout feature (#26)

### PR DESCRIPTION
**Pull request by MANISHIMWESalton**
* [#187584914]added logout feature

* [starts #187584914] added logout feature

* [finishes#187584914] logout feature

* [delivers##187584914] updated readme & swagger.json

* [delivers##187584914] updated readme & swagger.json

* [deliveres #187584914] logout features completed

* [deliveres #187584914] logout features completed

* [delivers #187584914] finished logout feature

### CODE REVIEW

**Review by SaddockAime — DISMISSED** [review]

**Review by Fabrice-Dush — DISMISSED** [review]

**Review by AimePazzo — APPROVED** [review]

**Review by SaddockAime — APPROVED** [review]

**Review by Jadowacu1 — APPROVED** [review]

**Review by Jadowacu1 — DISMISSED** [review]

**Review by AimePazzo — DISMISSED** [review]

**Review by SaddockAime — DISMISSED** [review]

File: src/routes/userRouter.ts [hunk 1]
```diff
@@ -12,4 +12,7 @@ router.get("/admin-get-user/:id",userAuthorization(["admin"]), isUserExist, user
 router.put("/admin-update-user-status/:id",userAuthorization(["admin"]), validation(statusSchema), isUserExist, userControllers.updateUserStatus);
 router.put("/admin-update-role/:id",userAuthorization(["admin"]), validation(roleSchema),isUserExist, userControllers.updateUserRole);
 
+router.get("/user-get-profile/:id",userAuthorization(["buyer", "seller"]),userControllers.getUserDetails)
+router.put("/user-update-profile/:id",userAuthorization(["buyer", "seller" ]),upload.single("profilePicture"),validation(userSchema), userControllers.updateUserProfile)
+
```
Comment (key-joshua) [inline src/routes/userRouter.ts:17]: We don't have to parse user id into parameters as we are getting a user id from userAuthorization middleware.

File: src/routes/userRouter.ts [hunk 2]
```diff
@@ -12,4 +12,7 @@ router.get("/admin-get-user/:id",userAuthorization(["admin"]), isUserExist, user
 router.put("/admin-update-user-status/:id",userAuthorization(["admin"]), validation(statusSchema), isUserExist, userControllers.updateUserStatus);
 router.put("/admin-update-role/:id",userAuthorization(["admin"]), validation(roleSchema),isUserExist, userControllers.updateUserRole);
 
+router.get("/user-get-profile/:id",userAuthorization(["buyer", "seller"]),userControllers.getUserDetails)
```
Comment (key-joshua) [inline src/routes/userRouter.ts]: We don't have to parse user id into parameters as we are getting a user id from userAuthorization middleware.

**Review by Fabrice-Dush — DISMISSED** [review]

**Review by SaddockAime — DISMISSED** [review]

File: src/databases/models/users.ts [hunk 3]
```diff
@@ -53,10 +53,10 @@ class Users
   declare createdAt?: Date;
   declare updatedAt?: Date;
 
-  // Define any static methods or associations here
-  static associate(models: any) {
-    Users.hasOne(models.Tokens, { foreignKey: "userId", as: "token" });
-  }
+    // Define any static methods or associations here
```
Comment (AimePazzo) [inline src/databases/models/users.ts:56]: I told you many times we don't merge with comment

**Review by AimePazzo — COMMENTED** [review]
Please remove all unnecessary comments

**Review by AimePazzo — APPROVED** [review]

**Review by SaddockAime — DISMISSED** [review]

**Review by SaddockAime — APPROVED** [review]

**Review by Fabrice-Dush — DISMISSED** [review]

**Review by SaddockAime — DISMISSED** [review]

**Review by Jadowacu1 — DISMISSED** [review]

**Pull request merged**